### DISCUSSION TITLE
Scaffold Generated wihtout any strucure and SMILES notation

### DISCUSSION THREAD

**Discussion by Crispae** [rdkit/rdkit#5256] [Q&A] [answered]
I have implemented **MurckoScaffoldSmilesFromSmiles** on the database to get a common scaffold present between them. Certain molecules have been grouped together with SMILES notation "". while others are giving legitimate scaffold structure. Empty SMILES notation does not define anything is it a bug or an implementation error?

code:
from rdkit.Chem.Scaffolds.MurckoScaffold import MurckoScaffoldSmilesFromSmiles
data["framework"] = data.Structure.apply(MurckoScaffoldSmilesFromSmiles)
![Capture](https://user-images.githubusercontent.com/74324530/166256576-cad56143-0f14-4672-8a61-3fbd9c0ba560.PNG)

**Comment by apahl** [comment, marked as answer]
Hi @Crispae,  
it would help to see some SMILES for the entries that resulted in empty scaffold SMILES, but I suspect, that these structures do not have any ring systems, which would lead to empty scaffolds.

KR Axel

**Reply by greglandrum** [reply]
I suspect @apahl is correct. Molecules without rings have empty Murcko scaffolds.
If those molecules *do* have rings, then please share them because something is wrong.

**Reply by Crispae** [reply]
Yes, These molecule doesn't hold any ring structure that´s why. Thanks for your help.
